### ISSUE TITLE
Setting Text property on TextBox results in Accessibility Insights check failure

### ISSUE THREAD

**Issue by chingucoding** [microsoft/microsoft-ui-xaml#3481]
Creating a TextBox with just the TextBox property set, Accessibility Insights reports a check failure:

![Image showing Accessibility Insights error](https://user-images.githubusercontent.com/16122379/97089384-94c6c900-1637-11eb-923b-f4e9a5644da2.png)

The XAML used:
```xaml
<TextBox Text="Some text"/>
```

Narrator announces the text of the control correctly though, so this might be an issue with Accessibility Insights. Creating an AutomationPeer for the TextBox through `FrameworkElement.CreateAutomationPeer` and calling `GetName` returns an empty string though, so I am not quite sure if this is intended or if I am missing something here.

In retrospective, this might make sense. The question that I have then is, what information should be returned in `FrameworkElementAutomationPeer.GetName`. Should things like the value for a progressbar be returned in there? Should textcontrols not return their text in there?

**Comment by ranjeshj** [comment]
GetName returning empty sounds reasonable. If you set AutomationProperties.Name, you would get it. If there is a header, you can get that as well. So everything seems fine from the control side. The element path issue sounds suspect and I wonder if it is an issue  with the rule itself.

Can we file this issue on the accessibility insights repo ?
https://github.com/Microsoft/accessibility-insights-windows

**Comment by chingucoding** [comment]
These might be the column headers for the failed rules below, created https://github.com/microsoft/accessibility-insights-windows/issues/980.

Since the behavior is expected in this case, I'll close this issue as there is nothing to track here then.
